src/NovelRT/Timing/StepTimer.cpp

```diff
@@ -8,7 +8,7 @@
 namespace NovelRT::Timing {
   StepTimer::StepTimer(uint32_t targetFrameRate, double maxSecondDelta) :
     _frequency(glfwGetTimerFrequency()),
-    _maxCounterDelta((uint64_t)(_frequency * maxSecondDelta)),
+    _maxCounterDelta((uint64_t)(_frequency* maxSecondDelta)),
```

Comment:
nit: a whitespace was removed

Reply:
i don't get why VS's autoformat wants it connected.